AK/HashMap.h

```diff
@@ -222,6 +222,17 @@ class HashMap {
         return find(key)->value;
     }
 
+    template<typename Callback>
+    ErrorOr<V> try_ensure(K const& key, Callback initialization_callback)
```

Comment:
the second commit is titled "add try_assert method` but it adds a method named "try_ensure" :)

Reply:
Fixed :)